NorthstarDLL/logging.cpp

```diff
@@ -209,6 +209,7 @@ long __stdcall ExceptionFilter(EXCEPTION_POINTERS* exceptionInfo)
 	}
 
 	logged = true;
+	reinterpret_cast<void (*)(char*)>((char*)GetModuleHandleA("engine.dll") + 0x118F00)((char*)GetModuleHandleA("engine.dll") + 0x12A53D40); // shut down server on crash.
```

Comment:
would probably be best if this wasn't called on every server death, and was instead only called if server was active, and if it hadn't already been called, could probably just set a bool here to track whether it's been called already and not call it after that, in case we crash inside server shutdown and get stuck in a crash loop